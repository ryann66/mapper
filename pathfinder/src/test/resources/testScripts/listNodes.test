CreateGraph g1
ListNodes g1
AddNode g1 bravo
ListNodes g1
#add to back
AddNode g1 delta
ListNodes g1
#add to front
AddNode g1 alpha
ListNodes g1
#add to middle
AddNode g1 charlie
ListNodes g1