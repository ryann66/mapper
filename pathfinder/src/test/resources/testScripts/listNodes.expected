created multigraph g1
g1 contains:
added node bravo to g1
g1 contains: bravo
#add to back
added node delta to g1
g1 contains: bravo delta
#add to front
added node alpha to g1
g1 contains: alpha bravo delta
#add to middle
added node charlie to g1
g1 contains: alpha bravo charlie delta